### PR TITLE
Quiz card

### PR DESCRIPTION
**Pull request by anniviz**

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -11,5 +11,67 @@
     <header class="app-header">
       <h1 class="app-header__heading">Quiz App</h1>
     </header>
+    <main class="app-main">
+      <section class="quiz-card">
+        <h2>Waht is the question?</h2>
```
Comment (macfrei) [inline src/index.html]: Just a small typo :)
```suggestion
        <h2>What is the question?</h2>
```

**Review by macfrei — COMMENTED** [review]
Looks good, really like your use of CSS variables!

**Review by macfrei — COMMENTED** [review]
Looks good! 👍

**Comment by anniviz** [comment]
Thanks!

**Pull request merged**